installer/install.iss

```diff
@@ -423,6 +423,7 @@ const
     GP_BuiltinAddI     = 4;
     GP_EnablePCon      = 5;
     GP_EnableFSMonitor = 6;
+    GP_Max             = 6;
```

Comment:
This is a great idea, but the name is slightly confusing since we use the same `GP_` prefix for other things.

Reply:
I revamped this rather completely. Could you have another look, @rimrul?